components/contribution-flow/StepDetailsCrypto.js

```diff
@@ -67,6 +112,16 @@ const StepDetailsCrypto = ({ onChange, data }) => {
         autoFocus
         error={touched && amount <= 0 && intl.formatMessage(messages['invalidAmount'])}
       />
+      {convertedAmount > 0 && (
```

Comment:
If `convertedAmount` is null, it's weird to test `convertedAmount > 0`.

Reply:
The reason I did it this way was because, suppose the user inserts -1 BTC; then the approximate amount will show as `~-$48,753.99 USD` for example. Now that seems quite weird to my eyes to show something that is like that to the user so I thought I'll hide the approximate amount completely if they enter a negative amount. 😉 

![crypto-negative](https://user-images.githubusercontent.com/12435965/131726454-01edecf4-c9e5-474c-85ad-b95dbff1c667.gif)

But I am willing to change this if you have a better suggestion on how to go about this? 🤔

Reply:
It's just not a good practice to do `null > 0`, there is a simple way to check that it's not null right?

As for negative, maybe you don't want to dispatch the value if it's invalid?

Reply:
> It's just not a good practice to do `null > 0`, there is a simple way to check that it's not null right?

Yes makes sense. I've done a short-circuit evaluation so that now it check for `null` first. 

> As for negative, maybe you don't want to dispatch the value if it's invalid?

Sure, that makes sense. Although I don't think calling the dispatch function does any harm as well (albeit it's useless) since the user cannot progress in the flow anyways since the button is disabled for negative values. But I've added a check to not call the `dispatchChage` function when it's negative.